environments/staging/vault.yml

```diff
@@ -1,38 +1,46 @@
-3733663439313166366539653263643031396264316263360a353364326632656333336630383061
-38343138306662323231643633646364623262326462646235396363656365366539396539663636
-3361666237316132340a643538323130343862373364343062633635653531626535376163643261
-32626562663233353739393562333438333532623261613339623232636463626366653463623335
-63366332373734393137653031303232396533666164643336613538633033633863626530366666
-61656632663938653431643433613631306239393432326564623135613463303463663434323738
-37653738663231653434356163666534306536626330336265666433393535633635316536353763
-38613937643634353735623161383932653164356563303261633735616562633534326338363532
-64636164646232306137363566633432656161393531363363306432353839646637643338366463
-61393061663530393465303332636261303132336564396663663132316165633165333939323530
-64646462393331343530336666663832383237306630336163326339313832633036383132656138
-39363533613539383932646164343433643364323261323563383630366435356231636230313964
-39333563646630643431346439313833333837316436363664613064663635653433363535626337
-30336138626531323032646663333031343862323562366435633536663531386533323064653638
-65323331356166366634343235633034623162643936633237396439373566316339376261393535
-63666162643436343639316130393866393166313738346439303561386331666664633538643166
-61353331353866656436356366333034633863386235343539393662646131396162326330363635
-34343861343066393630316565376337333438303731333632663533306439333166333665353764
-36653065666536613663353130663536653963393839656164333134343536353238383730656633
-38376262646432393030666630396236326232346330313637386239326330383033333339333562
-66623533316361663461626532633332363633623463306630643231633139393933643936323166
-33373961393266323233643562633035623630623564623738366361353036633931336439316439
-62623833623335356333633932326239616232323835656661353636636339313231656662383365
-32323861353536343165393664613430336566353263353032646130346464326334383863343862
-37313766393834393631653565356134643935363663363734626462613032373439616633306539
-31623637373130623036623939623962643837343037373063383038656434383338626262653337
-34333930336363376536373663323066613637393335653137316635636634356634616532323436
-62373464636162333662386230313537313532396161373631323264366531353131636233363461
-39363235313630623561323239643231323461613964636661353961383830313736623332333163
-33363238366365313339643735613133386265656332303430643136653238306164653634633835
-35356433383337653766643333333930666163336432643331363163653363306266613030386662
-31376337353166616133376362326431623736383034313838653939666134613335313662613537
-36656637386638623563313539656335633334623335653366346134326233366261656537346235
-38303131366566303035613364626436626530663237316234316563396331616464383935623262
-37623336626633393766616233613533373031313534393837376138353537343138393433396662
-3365
+36383163336537383366323966353632303235383433623538636464303362626433653730663064
```

Comment:
I haven't added it to the other environments since we don't do backups off those environments (truth be told, I don't even know if those environments are being used somewhere).

Reply:
Feel to remove the redundant environments @Charl1996 
Only analytics and staging are in use.

Reply:
looks like you missed adding for analytics one, that is production.

Reply:
Production runs from the `environments/production` environment as far as I know?

Reply:
Hey @Charl1996 

I am looking at this list, https://github.com/dimagi/commcare-analytics-ansible/tree/master/environments
I don't see production.

Reply:
The production env was created on install as a separate env. This is not ideal at the moment and we should probably push that to the repo as well (can remove `anayltics` env then).

Reply:
https://github.com/dimagi/commcare-analytics-ansible/pull/41